tutorials/01-Introduction.ipynb

```diff
@@ -0,0 +1,2872 @@
+    "\n",
+    "- Fibonacci number\n",
+    "- Factorials\n",
+    "- GCD\n",
+    "- Prime factorization\n",
+    "- ...\n",
+    "\n",
+    "which works with very large numbers"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "305e0477-1492-4d1b-9f79-d57881341c3e",
+   "metadata": {
+    "editable": true,
+    "raw_mimetype": "",
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "#### Fibonacci Numbers"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 61,
+   "id": "b0087f4e-a21c-4c22-b768-7b84120cd801",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "$\\displaystyle 573147844013817084101$"
+      ],
+      "text/plain": [
+       "573147844013817084101"
+      ]
+     },
+     "execution_count": 61,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "fibonacci(101)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 62,
+   "id": "526cf0bd-a355-459f-9eb4-a7a920a79c81",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "$\\displaystyle 70330367711422815821835254877183549770181269836358732742604905087154537118196933579742249494562611733487750449241765991088186363265450223647106012053374121273867339111198139373125598767690091902245245323403501$"
+      ],
+      "text/plain": [
+       "70330367711422815821835254877183549770181269836358732742604905087154537118196933579742249494562611733487750449241765991088186363265450223647106012053374121273867339111198139373125598767690091902245245323403501"
+      ]
+     },
+     "execution_count": 62,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "fibonacci(1001)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7ab4f831-7bed-42f4-9ba9-ac4b2689405d",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "#### Prime Factorization"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "97ec23e0-7437-4235-98fb-c4e374266a97",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "factorint(1234567890123456789)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "16111013-fb7f-476e-9cf4-967667627e01",
+   "metadata": {
+    "editable": false,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "## Rational Numbers"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "40bbdd9b-3cbb-43a6-9a59-b1cf43cf9a43",
+   "metadata": {},
+   "source": [
+    "### Why floating points are bad?"
```

Comment:
Floating point numbers aren't "bad". They just aren't appropriate for exact computation, which is what SymPy is designed for.